test/conformance/ingressv2/util.go

```diff
@@ -82,19 +82,11 @@ var dialBackoff = wait.Backoff{
 }
 
 var testGateway = &gatewayv1alpha1.RouteGateways{
-	Allow: gatewayAllowTypePtr(gatewayv1alpha1.GatewayAllowFromList),
-	GatewayRefs: []gatewayv1alpha1.GatewayReference{{
-		Namespace: "knative-serving",
-		Name:      "test-gateway",
-	}},
+	Allow: gatewayAllowTypePtr(gatewayv1alpha1.GatewayAllowAll),
```

Comment:
How come we aren't using GatewayAllowFromList anymore?

Reply:
Since Istio's Gateway needs to be deployed in `istio-system` namespace, Contour also needs to use the Gateway in `istio-system` namespace if we use `GatewayAllowFromList`. (That's so strange.)

I will fix it by https://github.com/knative-sandbox/net-ingressv2/issues/75 in a separate PR.